app/views/api_docs/reference/reference.html.erb

```diff
@@ -113,3 +117,22 @@
 <% @api_reference.schemas.each do |schema| %>
   <%= render 'schema', schema: schema %>
 <% end %>
+
+<h2 class="govuk-heading-l" id="field-lengths">Field lengths summary</h2>
```

Comment:
Can a link to this be added to the contents at the top of the page?

Reply:
Done!